EncounterMe/EncounterMe/Pins/MapPin.cs

```diff
@@ -52,7 +55,7 @@ public MapPin()
 
         public MapPin(string name, string description, Address address, int type = 0, int styleType = 0,
                        string open = "", string close = "", double latitude = 0, double longitude = 0,
-                       string image = "")
+                       string image = "", int? creatorId = null)
```

Comment:
O ką reiškia tas klaustukas, jau ankščiau norėjau to paklaust :D

Reply:
Kad kintamasis gali įgyt null reikšmę :D Paprastesnis tada tikrinimas visur kitur, ar gavom reikšmę ar ne